hudi-kafka-connect/src/main/java/org/apache/hudi/connect/utils/KafkaConnectUtils.java

```diff
@@ -85,9 +87,15 @@ public static int getLatestNumPartitions(String bootstrapServers, String topicNa
    *
    * @return
    */
-  public static Configuration getDefaultHadoopConf() {
+  public static Configuration getDefaultHadoopConf(KafkaConnectConfigs connectConfigs) {
     Configuration hadoopConf = new Configuration();
-    hadoopConf.set("fs.file.impl", org.apache.hadoop.fs.LocalFileSystem.class.getName());
+    connectConfigs.getProps().keySet().stream().filter(prop -> {
+      // In order to prevent printing unnecessary warn logs, here filter out the hoodie
+      // configuration items before passing to hadoop/hive configs
+      return !prop.toString().startsWith(HOODIE_CONF_PREFIX);
+    }).forEach(prop -> {
+      hadoopConf.set(prop.toString(), connectConfigs.getProps().get(prop.toString()).toString());
+    });
```

Comment:
I see similar code in `KafkaOffsetGen::excludeHoodieConfigs()`.  Could you keep one of them?

Reply:
I could, but i have to convert the kafka map configs to hadoop configs. also later we can change this logic if required. for instance, if we want to just have hadoop confs start with "conf.hadoop." in kafka connect, we can do that independently. wdyt?

Reply:
Got it.  Then let's keep this method.